xilem/examples/to_do_mvc.rs

```diff
@@ -6,9 +6,13 @@
 // On Windows platform, don't show a console when opening the app.
 #![windows_subsystem = "windows"]
 
+use masonry::properties::types::Length;
+use masonry::theme::{DEFAULT_GAP, ZYNC_800};
```

Comment:
```suggestion
use xilem::masonry::properties::types::Length;
use xilem::masonry::theme::{DEFAULT_GAP, ZYNC_800};
```

As I think users should not have to add `masonry` as direct dependency.